lib/chef/win32/api/file.rb

```diff
@@ -565,6 +600,21 @@ def retrieve_file_info(file_name)
           file_information
         end
 
+        def retrieve_file_version_info(file_name)
+          file_name = encode_path(file_name)
+          file_size = GetFileVersionInfoSizeW(file_name, "")
```

Comment:
The second parameter here needs to be a pointer to a DWORD according to https://msdn.microsoft.com/en-us/library/windows/desktop/ms647005%28v=vs.85%29.aspx